Func/Convert.cs

```diff
@@ -18,7 +18,7 @@ public static class Convert
         //
         [FunctionName("Convert")]
 #pragma warning disable VSTHRD200 // Use "Async" suffix for async methods - Name must be "Run" for this to work AFAIK
-        public static async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req, ILogger log)
+        public static async Task<IActionResult> Run([HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = null)] HttpRequest req, ILogger log)
```

Comment:
@christophwille I was a bit confused when I came across this. It looks like it was currently a get request that has a body (which is weird). But even weirder is that the deployed version actually 404s for "GET", but works for "POST". So I don't think I'm changing anything, but just wanted to flag this in case I've misunderstodd what's going on here

Example url I tried with get/post:
https://dotnet-codeconverter.azurewebsites.net/api/Convert

Reply:
POST was very intentional because of size limitations on GET via URL (and caching of proxies in between)

Reply:
Ah phew, nothing weird going on here. It looks like it was accidentally changed to `GET` by my commit just before - I think I must have copy pasted when suppressing the naming issue and not noticed the difference 😬 All fixed now 😃